test/e2e/flowaggregator_test.go

```diff
@@ -1865,6 +1879,65 @@ func getAndCheckFlowAggregatorMetrics(t *testing.T, data *TestData) error {
 	return nil
 }
 
+func testL7FlowExporterController(t *testing.T, data *TestData, isIPv6 bool) {
+	skipIfFeatureDisabled(t, features.L7FlowExporter, true, false)
+
+	err := data.UpdatePod(data.testNamespace, "perftest-a", func(pod *corev1.Pod) {
+		pod.Annotations = map[string]string{antreaagenttypes.L7FlowExporterAnnotationKey: "both"}
+	})
+	require.NoErrorf(t, err, "error when updated pod annotations")
+
+	testFlow1 := testFlow{
+		srcPodName: "perftest-a",
+		dstPodName: "perftest-b",
+	}
+	var cmd []string
+	if !isIPv6 {
+		testFlow1.srcIP = podAIPs.IPv4.String()
+		testFlow1.dstIP = podBIPs.IPv4.String()
+		cmd = []string{
+			"curl",
```

Comment:
I think curl is not supported in perftest Pods. Maybe use a nginx Pod?

Reply:
I think you are still using toolboxImage for the clientPod which doesn't have curl installed inside?

Reply:
antrea/toolbox is the correct image to use for "client" Pods as a rule, and it does include curl
"server" Pods will typically use nginx or agnhost based on the test needs

Reply:
Which version are you referring to? I think antrea/toolbox:1.2-1 doesn't support curl.

Reply:
mb, I thought perftestPods are still using perftool.